docs/index.rst

```diff
@@ -405,7 +405,7 @@ Use the following tokens for parsing and formatting. Note that they are **not**
 .. [#t1] localization support for parsing and formatting
 .. [#t2] localization support only for formatting
 .. [#t3] the result is truncated to microseconds, with `half-to-even rounding <https://en.wikipedia.org/wiki/IEEE_floating_point#Roundings_to_nearest>`_.
-.. [#t4] timezone names from `tz database <https://www.iana.org/time-zones>`_ provided via dateutil package
+.. [#t4] timezone names from `tz database <https://www.iana.org/time-zones>`_ provided via dateutil package, note that abbreviations such as MST, PDT, BRST are unlikely to parse due to being ambiguous.
```

Comment:
Looks good, but do you think we should add an extra sentence to explain that one should use the full timezone expression rather than the abbreviation for this to work? Also we should probably change "being ambiguous" to "ambiguity".